docs/reference/configuration.txt

```diff
@@ -131,100 +131,84 @@ for details on driver options.
   development:
     # Configure available database clients. (required)
     clients:
-      # Define the default client. (required)
+      # Defines the default client. (required)
       default:
-        # A uri may be defined for a client:
-        # uri: 'mongodb://user:password@myhost1.mydomain.com:27017/my_db'
-        # Please see driver documentation for details. Alternatively, you can
-        # define the following:
-        #
-        # Define the name of the default database that Mongoid can connect to.
+        # Mongoid can connect to a URI accepted by the driver:
+        # uri: mongodb://user:password@mongodb.domain.com:27017/my_db_development
+
+        # Otherwise define the parameters separately.
+        # This defines the name of the default database that Mongoid can connect to.
         # (required).
-        database: my_db
-        # Provide the hosts the default client can connect to. Must be an array
+        database: my_db_development
+        # Provides the hosts the default client can connect to. Must be an array
         # of host:port pairs. (required)
         hosts:
-          - myhost1.mydomain.com:27017
-          - myhost2.mydomain.com:27017
-          - myhost3.mydomain.com:27017
+          - localhost:27017
         options:
-          # These options are Ruby driver options, documented in
-          # https://mongodb.com/docs/ruby-driver/current/reference/create-client/
-
+          # Note that all options listed below are Ruby driver client options (the mongo gem).
+          # Please refer to the driver documentation of the version of the mongo gem you are using
+          # for the most up-to-date list of options.
+          #
```

Comment:
World remove the `#` on this line to separate the above general comment from write concern below.